deploy/bootstrap.py

```diff
@@ -23,13 +23,222 @@
     check_call,
     get_conda_base,
     get_logger,
-    get_spack_base,
     install_miniforge,
     parse_args,
 )
 
 
-def get_config(config_file, machine):
+def main():  # noqa: C901
+    args = parse_args(bootstrap=True)
+    options = vars(args)
+    options['conda_env_name'] = options['env_name']
```

Comment:
Let's just rename `env_name` to `conda_env_name` in `args` as well.  The flag can still be `--env_name` but you can have `dest=conda_env_name`:
https://github.com/E3SM-Project/polaris/blob/main/deploy/shared.py#L20

Reply:
This is easier than dropping `env_name` which would be an alternative.  I think we don't want to have 2 options that are initially the same but could potentially deviate from each other (e.g. when `conda_env_name` gets set but `env_name` remains with the old value of `None`).

Reply:
Sounds good, will do.